src/coreclr/pal/src/synchobj/mutex.cpp

```diff
@@ -529,39 +577,76 @@ OpenMutexA (
 Function:
   OpenMutexW
 
+Parameters:
+  See doc for PAL_OpenMutexW.
+--*/
+
+HANDLE
+PALAPI
+OpenMutexW(
+       IN DWORD dwDesiredAccess,
+       IN BOOL bInheritHandle,
+       IN LPCWSTR lpName)
+{
+    return PAL_OpenMutexW(dwDesiredAccess, bInheritHandle, lpName, nullptr, 0);
+}
+
+/*++
+Function:
+  PAL_OpenMutexW
+
 Note:
   dwDesiredAccess is currently ignored (no Win32 object security support)
   bInheritHandle is currently ignored (handles to mutexes are not inheritable)
 
-See MSDN doc.
+Parameters:
+  lpSystemCallErrors -- An optional buffer into which system call errors are written, for more detailed error information.
+  dwSystemCallErrorsBufferSize -- Size of the buffer pointed to by lpSystemCallErrors in bytes.
+
+  See MSDN docs on OpenMutexW for all other parameters.
 --*/
 
 HANDLE
 PALAPI
-OpenMutexW(
+PAL_OpenMutexW(
        IN DWORD dwDesiredAccess,
```

Comment:
We can omit the `dwDesiredAccess`and `bInheritHandle` arguments from the `PAL_` version. They are not used.